packages/core/src/endowmentsToolkit.js

```diff
@@ -15,14 +15,28 @@
  * @packageDocumentation
  */
 
+/**
+ * WARNING: This module is used directly by the runtime in webpack plugin which
+ * uses simple shimming to assemble modules. It doesn't bundle properly. This
+ * file cannot reqire any files or packages.
+ */
```

Comment:
```suggestion
 * WARNING: This file may not require any files or packages.
 * This module is used directly by the runtime in webpack plugin which
 * uses simple shimming to assemble modules and does not bundle properly. 
 */

```


 Nit: Putting the actionable message first for clarity (also there was a small typo). 

Aside: If this turns out to be more than just transient, should look at having an extra linting rule for this file/module to catch it IMO